packages/service-integration/setup.py

```diff
@@ -0,0 +1,51 @@
+import re
+import sys
+from pathlib import Path
+
+from setuptools import find_packages, setup
+
+here = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
+
+
+def read_reqs(reqs_path: Path):
+    return re.findall(r"(^[^#-][\w]+[-~>=<.\w]+)", reqs_path.read_text(), re.MULTILINE)
+
+
+install_requirements = read_reqs(here / "requirements" / "_base.in") + [
+    "simcore-models-library"
+]  # WEAK requirements
+
+test_requirements = read_reqs(
+    here / "requirements" / "_test.txt"
+)  # STRONG requirements
```

Comment:
What do the `WEAK` and `STRONG` refer to?

Reply:
perhaps it is better to say STRICT requirements, i.e. those that are pip-compiled and are frozen.
When it comes to libraries, you tend to be more flexible in the versions so that multiple libraries can be easily installed in the same environ